java/vector/pom.xml

```diff
@@ -32,6 +32,11 @@
       <artifactId>arrow-memory-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-netty</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
add a comment why this dependency is needed

Reply:
Done here https://github.com/sirensolutions/arrow/pull/16/commits/5d79a1902316072bb4b05250736b1fe669182141